pkg/serverless/trace/cold_start_span_creator.go

```diff
@@ -46,6 +46,7 @@ type ColdStartSpanCreator struct {
 	initDuration          float64
 	StopChan              chan struct{}
 	initStartTime         time.Time
+	WasColdStart          bool
```

Comment:
Can you add a doc comment here? I'm still confused exactly what this field means. My naive understanding is that this will always be true.

Reply:
yeah I'll revisit this and make it clearer with a comment